draft-kazuho-httpbis-priority.md

```diff
@@ -98,6 +98,62 @@ The terms sh-token and sh-boolean are imported from
 Example HTTP requests and responses use the HTTP/2-style formatting from
 {{?RFC7540}}.
 
+# Negotiating Priorities
+
+The HTTP/2 specification defines a priority scheme in [RFC7540],
+Section 5.3, which some implementers have opted not to fully support.
+The lack of signalling about the status of the implementation has
+caused several implementations to implement heuristics to detect when
+the clients they are connected to do not support priorities as
+defined and take steps to compensate for that.  The intent of this
+negotiation is to provide an affirmative signalling mechanism for each
+peer to communicate which, if any, priority schemes are supported,
```

Comment:
the intent of this negotiation is also to allow the peers to specify their ranked preferences of prioritization schemes, so we should note that here.